### ISSUE TITLE
Pagefind issue when installing as remote theme.

### ISSUE THREAD

**Issue by gdtroszak** [lumeland/theme-simple-blog#9]
Hi - I really love lume and the simplicity of this template!

I followed the instructions for [Install as remote theme](https://github.com/lumeland/theme-simple-blog#install-as-remote-theme) and hit the following the error when running `deno task serve`:

```bash
Task serve deno task lume -s
Task lume echo "import 'lume/cli.ts'" | deno run --unstable -A - "-s"
Loading config file file:///REPO_DIR/_config.ts


🔥 /readme/ /README.md
🔥 /feed.json https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/feed.tmpl.js
🔥 /404.html https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/404.md
🔥 /styles.css https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/styles.css
🔥 /feed.xml https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/feed.xml.njk
🔥 / https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/index.njk
🔥 /sitemap.xml (generated)
🔥 /robots.txt (generated)
🔥 /favicon.png https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/favicon.png
🔥 /fonts/epilogue-bold.woff2 https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/fonts/epilogue-bold.woff2
🔥 /fonts/inter-italic.woff2 https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/fonts/inter-italic.woff2
🔥 /fonts/inter.woff2 https://deno.land/x/lume_theme_simple_blog@v0.5.1/src/fonts/inter.woff2

Using binary file at /Users/gregtroszak/development/gregtroszak.me/_bin/pagefind
Error: Pagefind exited with code 127



47,85,115,101,114,115,47,103,114,101,103,116,114,111,115,122,97,107,47,100,101,118,101,108,111,112,109,101,110,116,47,103,114,101,103,116,114,111,115,122,97,107,46,109,101,47,95,98,105,110,47,112,97,103,101,102,105,110,100,58,32,108,105,110,101,32,49,58,32,51,48,58,32,99,11
1,109,109,97,110,100,32,110,111,116,32,102,111,117,110,100,10,47,85,115,101,114,115,47,103,114,101,103,116,114,111,115,122,97,107,47,100,101,118,101,108,111,112,109,101,110,116,47,103,114,101,103,116,114,111,115,122,97,107,46,109,101,47,95,98,105,110,47,112,97,103,101,102,1
05,110,100,58,32,108,105,110,101,32,50,58,32,50,50,58,32,99,111,109,109,97,110,100,32,110,111,116,32,102,111,117,110,100,10,47,85,115,101,114,115,47,103,114,101,103,116,114,111,115,122,97,107,47,100,101,118,101,108,111,112,109,101,110,116,47,103,114,101,103,116,114,111,115,
122,97,107,46,109,101,47,95,98,105,110,47,112,97,103,101,102,105,110,100,58,32,108,105,110,101,32,51,58,32,50,50,58,32,99,111,109,109,97,110,100,32,110,111,116,32,102,111,117,110,100,10,47,85,115,101,114,115,47,103,114,101,103,116,114,111,115,122,97,107,47,100,101,118,101,1
08,111,112,109,101,110,116,47,103,114,101,103,116,114,111,115,122,97,107,46,109,101,47,95,98,105,110,47,112,97,103,101,102,105,110,100,58,32,108,105,110,101,32,52,58,32,50,56,58,32,99,111,109,109,97,110,100,32,110,111,116,32,102,111,117,110,100,10,47,85,115,101,114,115,47,1
03,114,101,103,116,114,111,115,122,97,107,47,100,101,118,101,108,111,112,109,101,110,116,47,103,114,101,103,116,114,111,115,122,97,107,46,109,101,47,95,98,105,110,47,112,97,103,101,102,105,110,100,58,32,108,105,110,101,32,53,58,32,51,50,58,32,99,111,109,109,97,110,100,32,11
0,111,116,32,102,111,117,110,100,10

    at (https://deno.land/x/lume@v1.15.0/plugins/pagefind.ts:185:15)
    at async Events.dispatchEvent (https://deno.land/x/lume@v1.15.0/core/events.ts:49:11)
    at async Site.build (https://deno.land/x/lume@v1.15.0/core/site.ts:527:5)
    at async build (https://deno.land/x/lume@v1.15.0/cli/build.ts:36:3)
    at async Command.execute (https://deno.land/x/cliffy@v0.25.7/command/command.ts:1794:7)
    at async Command.parseCommand (https://deno.land/x/cliffy@v0.25.7/command/command.ts:1639:14)
    at async (https://deno.land/x/lume@v1.15.0/cli.ts:158:3)
```

This appears to be an issue with pagefind? But figured I'd report here since this is where I hit it. Feel free to close if there's nothing you can do about it.

**Comment by gdtroszak** [comment]
FWIW, I ran the above using `lume v1.15.0`. When I downgrade to `lume v1.14.2`, all is well.

**Comment by gdtroszak** [comment]
FWIW, explicitly downgrading `pagefind` from `v0.10.6` to `v0.10.5` fixes the issue (which may mean there's not much that can be done here?)

```ts
...
.use(pagefind({ binary: { version: "v0.10.5" } }))
...
```

**Comment by oscarotero** [comment]
Hi. I remember to get a similar error at some point and after removing the `_bin` folder (so the binary file is downloaded again) it worked fine.
When you downgraded the version, the binary was downloaded again and maybe this is why it works. Can you try to update it to `v0.10.6` again?

**Comment by gdtroszak** [comment]
Yep, exactly. For my own sanity I:

1. Removed the `_bin` folder (containing `pagefind v0.10.5`.
2. Explicitly configured the plugin to use `pagefind v0.10.6`.
3. Ran `deno task serve`.

That resulted in the error I initially reported, so it definitely seems to be related to pagefind @ 0.10.6.

**Comment by oscarotero** [comment]
It's weird, because I'm using the last version in the Lume blog (https://github.com/lumeland/blog) without problems.

**Comment by gdtroszak** [comment]
> It's weird, because I'm using the last version in the Lume blog (https://github.com/lumeland/blog) without problems.

Very strange. I just cloned the blog repo and ran into the same issue...

**Comment by oscarotero** [comment]
Mmmm, what's your macOS? M1 or Intel processor?

**Comment by gdtroszak** [comment]
M1 running Ventura 13.1.

**Comment by oscarotero** [comment]
I'm running Intel. Maybe it's a bug in the M1 version. Lume downloads the binary according to your OS and architecture from here: https://github.com/CloudCannon/pagefind/releases/tag/v0.10.6

**Comment by gdtroszak** [comment]
I think that's it.

I just tried running the barebones example in an Alpine Docker container and got past the issue...

**Comment by bglw** [comment]
NB: Looks like this will have been fixed in lumeland/lume#362 / Pagefind 0.10.7
